about.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <div class="mobile-menu">
+    <div class="mobile-menu-background">
+      <img src="images/Iconcross-btn.png" alt="Closing button mobile" class="cross-button pointer">
+    </div>
+    <ul class="mobile-menu-list">
+      <li class="mobile-menu-listItems"><a href="about.html">About</a></li>
+      <li class="mobile-menu-listItems"><a href="index.html#main-program">Program</a></li>
+      <li class="mobile-menu-listItems"><a href="index.html#clubs">Clubs</a></li>
+      <li class="mobile-menu-listItems"><a href="#partners">Partners</a></li>
+      <li class="mobile-menu-listItems"><a href="#contact">Contact</a></li>
+    </ul>
```

Comment:
- [ ] Kindly add "Home" link at the "About page" to  possible to navigate between the pages, there's a link from Home page to About page and the opposite too.

![Capture](https://user-images.githubusercontent.com/93598189/203608718-a1693730-c980-4ecf-9cda-6b8be89e5aec.PNG)